locations/spiders/hobby_lobby.py

```diff
@@ -0,0 +1,44 @@
+from urllib.parse import urlparse
+
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class HobbyLobbySpider(SitemapSpider, StructuredDataSpider):
+    name = "hobby_lobby"
+    allowed_domains = ["hobbylobby.com"]
+    item_attributes = {
+        "brand": "Hobby Lobby",
+        "brand_wikidata": "Q5874938",
+    }
+    sitemap_urls = ["https://www.hobbylobby.com/sitemap-Store.xml"]
+    sitemap_rules = [
+        (r"/stores/search/(\d+)$", "parse_sd"),
+    ]
+    time_format = "%H:%M %p"
+    custom_settings = {
+        # Site has no rate limiting
+        "DOWNLOAD_DELAY": 0.1,
```

Comment:
Generally use of `DOWNLOAD_DELAY` within spiders is reserved for increasing the download delay from a default value to avoid triggering a rate limit. If someone running the spider (or many spiders) wants to speed things up, they'd set the default `DOWNLOAD_DELAY` via settings.py to be a smaller value than the current default of `1` (1 second).

This custom setting should thus be removed from the spider.